Formula/nauty.rb

```diff
@@ -55,7 +56,7 @@ def install
         return 0;
       }
     EOS
-    system ENV.cc, "test.c", "-I#{include}", "-L#{lib}", "-lnauty", "-o", "test"
+    system ENV.cc, "test.c", "-I#{include}/nauty", "-L#{lib}", "-lnauty", "-o", "test"
```

Comment:
Is `-I#{include}` sufficient?

Reply:
Not now that `nauty.h` is in `/usr/local/nauty`. That said, I can instead change the test code to `#include <nauty/nauty.h>`. Is there a specific preference for Homebrew formulae?

Reply:
not really. I will take this :)